reporters/reporter.go

```diff
@@ -77,6 +84,8 @@ func Report(event string, opts map[string]interface{}) error {
 
 // MakeReporters creates Reporters' singleton from config/{}.yaml
 func MakeReporters(config *viper.Viper, logger logrus.FieldLogger) {
+	GetInstance().setLogger(logger)
```

Comment:
Since the `Reporters` is a Singleton, I needed to add this function to be able to log the `Report` errors.